tests/test_world_clock.py

```diff
@@ -0,0 +1,215 @@
+from app.routers.world_clock import generate_possible_timezone_path,\
```

Comment:
Prefer splitting using parenthesis (apply this rule also for the rest of the code)

Reply:
manage to do it except one...